tests/parsers/fixtures/pw/default_xml_220603/data-file-schema.xml

```diff
@@ -26,7 +26,7 @@
       <stress>true</stress>
       <forces>true</forces>
       <wf_collect>true</wf_collect>
-      <disk_io>low</disk_io>
+      <disk_io>nowf</disk_io>
```

Comment:
As you mentioned in the corresponding issue, this change as a means to test the changes in the schema is perhaps not the best. If we really feel we need to test this, we add a dedicated test that takes an example output XML for each schema version, then parametrizes it over the valid values of `disk_io` and just calls the schema validation. So completely decouple it from the parser tests.

But I am not sure this is even needed as now we are testing the validity of the schema, which should be static. So I think that if you run an example calculation locally with these changes and disk_io set to nowf and it works, then we accept that without specific unit tests.
```suggestion
      <disk_io>low</disk_io>
```